### ISSUE TITLE
Heroes and Citizens conflicting.

### ISSUE THREAD

**Issue by xxrepresent** [CitizensDev/Citizens#670]
This plugin is good, but loses it ability to becoming more than just a cool add on because of it's conflicts with heroes. I assume that developers are already working on it, but im submitting this ticket to reinforce the issue.

**Comment by gummby8** [comment]
This conflict with Heros NEEDS to get fixed. THe potential that both plugins provide together is unparalleled to any others.

I have already seen citizens crash entirely on occasion due to projectiles, but when used with Heros it crashes 100% of the time any NPC is killed by arrow, egg, or snowball

**Comment by xxrepresent** [comment]
I agree completely with you. So many things could be accomplished if they worked! For now, I think just setting it so npcs cant be attacked makes it compliable. Haven't tested though.
